diff --git a/pico-keys-sdk/src/usb/usb_descriptors.c b/pico-keys-sdk/src/usb/usb_descriptors.c
index c44b3e2..f2f2734 100644
--- a/pico-keys-sdk/src/usb/usb_descriptors.c
+++ b/pico-keys-sdk/src/usb/usb_descriptors.c
@@ -214,7 +214,7 @@ enum
   VENDOR_REQUEST_WEBUSB = 1,
   VENDOR_REQUEST_MICROSOFT = 2
 };
-#define URL  "www.picokeys.com"
+#define URL  "www.github.com/librekeys"
 static bool web_serial_connected = false;
 
 const tusb_desc_webusb_url_t desc_url =
@@ -318,7 +318,7 @@ uint8_t const *tud_descriptor_bos_cb(void) {
 // array of pointer to string descriptors
 char const *string_desc_arr [] = {
     (const char[]) { 0x09, 0x04 }, // 0: is supported language is English (0x0409)
-    "Pol Henarejos",                     // 1: Manufacturer
+    "Libre Keys",                      // 1: Manufacturer
     "Pico Key",                       // 2: Product
     "11223344",                      // 3: Serials, should use chip ID
     "Config"               // 4: Vendor Interface
